release/src/Main.hs

```diff
@@ -50,7 +50,26 @@ main = do
       let nonDeployJars = filter (not . isDeployJar . artReleaseType) mavenUploadArtifacts
       let allMavenTargets = Set.fromList $ fmap (T.unpack . getBazelTarget . artTarget) mavenUploadArtifacts
 
-      -- first find out all the missing internal dependencies
+      -- check that all maven artifacts use com.daml as groupId
+      let nonComDamlGroupId = filter (\a -> ("com.daml" /=) $ foldr (<>) "" (List.intersperse "." $ map T.unpack $ pomGroupId $ artMetadata a)) mavenUploadArtifacts
```

Comment:
```suggestion
      let nonComDamlGroupId = filter (\a -> "com.daml" /= List.intercalate "." (map T.unpack $ pomGroupId $ artMetadata a)) mavenUploadArtifacts
```
Please don’t ask me how you should remember the difference between `intercalate` and `intersperse`. I just randomly try both until it works.

Reply:
Thanks. I mostly copy-pasta-ed this from some code further below. Wasn't super happy with it, but also none the wiser. I'll fix it there as well.